packages/chevrotain/src/parse/parser/parser.ts

```diff
@@ -265,9 +268,8 @@ export class Parser {
       )
     }
 
-    this.skipValidations = has(config, "skipValidations")
```

Comment:
I think the `has` check was intentional here.
I think we should ask ourselves:

In what way should we deal with configs/options of the **public API**
   - e.g: missing properties vs undefined properties, optional types vs `| undefined`

Then when we make a decision, apply this style **consistently** over the **whole** code base
in a a **separate** PR

Reply:
I kept the _has and used a `as boolean` cast to avoid (minor) possible semantic changes.